.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you please remove these eclipse specific files? Otherwise this pr looks good!